api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-3.0.x.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<databaseChangeLog
+	xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog 
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+	<!--
+    The empty change set is needed for integration tests. It can be removed if this file contains other change sets.
+-->
+	<changeSet id="42ddb873-f064-4418-a6c0-6c09426c4832" author="wschlegel">
+		<comment>Empty change set for integration tests.</comment>
```

Comment:
Do you still need this?

Reply:
Let me put it differently, if i added an additional changeset with `dbms="postgresql"`, would that be enough to remove the empty changeset?

Reply:
Updated the comment

Reply:
The test will pass on the default setting, but it will still fail if someone runs it on MySQL or PostgreSQL. That’s why we need a database-independent change set.